README.md

```diff
@@ -38,174 +38,117 @@ Function        Set-CloudEventJsonData                             0.2.0      Cl
 Function        Set-CloudEventXmlData                              0.2.0      CloudEvents.Sdk
 ```
 
-## Event Producer
+## Using **`CloudEvents.Sdk`** Module
+## 1. Event Producer
 ### Create a CloudEvent object
 ```powershell
-$cloudEvent = New-CloudEvent -Type 'com.example.object.deleted.v2' -Source 'mailto:cncf-wg-serverless@lists.cncf.io' -Id '6e8bc430-9c3a-11d9-9669-0800200c9a66' -Time (Get-Date)
+$cloudEvent = New-CloudEvent -Type 'com.example.object.deleted.v2' -Source 'mailto:cncf-wg-serverless@lists.cncf.io' -Id (New-Guid).Guid -Time (Get-Date)
 ```
 
-### Add **JSON Data** to a CloudEvent object
+### Set **JSON Data** to a CloudEvent object
 ```powershell
-$cloudEvent | Add-CloudEventJsonData -Data @{
-   'key1' = 'value1'
-   'key2' = @{
-      'key3' = 'value3'
-   }
+$cloudEvent | Set-CloudEventJsonData -Data @{
+    'Foo' = 'Hello'
+    'Bar' = 'World'
 }
+
+
+DataContentType : application/json
+Data            : {
+                    "Bar": "World",
+                    "Foo": "Hello"
+                  }
+Id              : ac9b12d9-ae45-4654-a4d7-42bbf0d9816d
+DataSchema      :
+Source          : mailto:cncf-wg-serverless@lists.cncf.io
+SpecVersion     : V1_0
+Subject         :
+Time            : 4/26/2021 9:00:45 AM
+Type            : com.example.object.deleted.v2
 ```
 
-### Add **XML Data** to a CloudEvent object
+### Set **XML Data** to a CloudEvent object
 ```powershell
-$cloudEvent | Add-CloudEventXmlData -Data @{
-   'key1' = @{
-      'key2' = 'value'
-   }
+$cloudEvent | Set-CloudEventXmlData -Data @{
+    'xml' = @{
+        'Foo' = 'Hello'
+        'Bar' = 'World'
+    }    
 } `
 -AttributesKeysInElementAttributes $true
-```
-`AttributesKeysInElementAttributes` specifies how to format the XML. If `true` and the input Data hashtable has pairs of 'Attributes', 'Value' keys creates XML element with attributes, otherwise each key is formatted as XML element.<br/>
-If `true`
-```powershell
-   @{'root' = @{'Attributes' = @{'att1' = 'true'}; 'Value' = 'val-1'}}
-```
-is formatted as
-```xml
-<root att1="true">val-1</root>
-```
-If `false`
-```powershell
-@{'root' = @{'Attributes' = @{'att1' = 'true'}; 'Value' = 'val-1'}}
-```
-is formatted as
-```xml
-<root><Attributes><att1>true</att1></Attributes><Value>val-1</Value></root>
-```
 
+
+DataContentType : application/xml
+Data            : <xml><Bar>World</Bar><Foo>Hello</Foo></xml>
+Id              : ac9b12d9-ae45-4654-a4d7-42bbf0d9816d
+DataSchema      :
+Source          : mailto:cncf-wg-serverless@lists.cncf.io
+SpecVersion     : V1_0
+Subject         :
+Time            : 4/26/2021 9:00:45 AM
+Type            : com.example.object.deleted.v2
+```
 #### Add Custom Format Data to a CloudEvent object
```

Comment:
Is this supposed to live under the XML example (heading level)?
Can we also provide an example output as we do for JSON/XML above?

Reply:
No, it's supposed to be separate.
Adding the output